repairnator/scripts/launch_dockerpool.sh

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash
+set -e
+
+# Use to create args in the command line for optional arguments
+function ca {
+  if [ -z "$2" ];
+  then
+      echo ""
+  else
+    if [ "$2" == "null" ];
+    then
+        echo ""
+    else
+        echo "$1 $2 "
+    fi
+  fi
+}
+
+if [ "$#" -eq 0 ]; then
+    echo "A list of build ids must be provided"
```

Comment:
Put a usage at the beginning of the script. See what I did in `repair_buggy_build.sh`

Reply:
I will do it.